src/pc/stage_sel.c

```diff
@@ -0,0 +1,141 @@
+#include <game.h>
+#include "../st/sel/sel.h"
+#include "pc.h"
+#include "sfx.h"
+
+// stubs
+RECT D_80182584 = {0};
+RECT D_8018258C = {0};
+RECT D_801825A4 = {0};
+void* D_8018C404[100] = {NULL};
+u16 D_800737B0; // g_Entities[5].step
```

Comment:
If these are actually g_Entities[5].step, isn't that going to be an issue since updates to one won't affect the other?